xen/drivers/passthrough/device_tree.c

```diff
@@ -187,7 +187,7 @@ int iommu_do_dt_domctl(struct xen_domctl *domctl, struct domain *d,
         ret = iommu_deassign_dt_device(d, dev);
 
         if ( ret )
-            printk(XENLOG_G_ERR "XEN_DOMCTL_assign_dt_device: assign \"%s\""
+            printk(XENLOG_G_ERR "XEN_DOMCTL_deassign_dt_device: assign \"%s\""
                    " to dom%u failed (%d)\n",
```

Comment:
I have updated pull request for your convenience. Now it contains patches that need to be reviewed only.